contracts/governance/Governor.sol

```diff
@@ -327,6 +327,20 @@ abstract contract Governor is Context, ERC165, EIP712, IGovernor, IERC721Receive
         return proposalId;
     }
 
+    /**
+     * @dev See {IGovernorTimelock-queue}.
+     *
+     * Note: This is an empty declaration for timelock modules to hook into.
+     */
+    function queue(
+        address[] memory /*targets*/,
+        uint256[] memory /*values*/,
+        bytes[] memory /*calldatas*/,
+        bytes32 /*descriptionHash*/
+    ) public virtual returns (uint256) {
+        revert();
```

Comment:
Why not a `NonImplementedMethod` error or similar?

Reply:
That would be ok with me.

Reply:
updated to `GovernorQueueNotImplemented`